_data/projects/home-assistant.io.yml

```
@@ -9,7 +9,7 @@ tags:
 - hass
 - hassio
 upforgrabs:
-  name: hacktoberfest
+  name: help wanted
```

This is case-sensitive:

```suggestion
  name: Help wanted
```